src/Exceptionless.Core/Configuration/MessageBusOptions.cs

```diff
@@ -22,12 +22,27 @@ public static MessageBusOptions ReadFromConfiguration(IConfiguration config, App
         options.Topic = config.GetValue<string>(nameof(options.Topic), $"{options.ScopePrefix}messages")!;
 
         string? cs = config.GetConnectionString("MessageBus");
-        options.Data = cs.ParseConnectionString();
-        options.Provider = options.Data.GetString(nameof(options.Provider));
+        if (cs != null)
```

Comment:
Really wonder if we should just gut our provider-based connection strings only thing it does is make it more pluggable and I guess is nice for aws vs azure vs other but the complexity. Especially https://github.com/exceptionless/Exceptionless/issues/1526

Reply:
Not getting into any more changes in this PR. These config changes were made so that the Aspire dashboard would work with the OTEL env variables it passes into the app.